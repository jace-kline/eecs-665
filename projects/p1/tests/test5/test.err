FATAL [1,78]: Illegal character \
