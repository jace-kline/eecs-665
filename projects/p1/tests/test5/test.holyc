"This is a test of escaping inside a string literal with an additional space \ n Hello testing testing"