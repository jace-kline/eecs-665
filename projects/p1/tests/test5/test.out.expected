EOF [1,104]